src/components/Explore/DateMarker.tsx

```diff
@@ -15,9 +15,9 @@ const DateMarker: React.FC<{
   seriesList: Series[];
   date: Date;
 }> = ({ left, seriesList, date }) => {
-  const [, seriesSmooth] = seriesList;
-  const pointSmooth = findPointByDate(seriesSmooth.data, date);
-  const dateToCompareTo = pointSmooth ? getColumnDate(pointSmooth) : date;
+  const series = seriesList[0];
```

Comment:
I'm not sure why we were using the second (`seriesSmooth`) series before.  All of the series should have their dates aligned, right?  Else the chart is going to be "wrong".  In any case I had to change it to use the first series because when I filtered out empty series, we could end up with a `seriesList` of length 1 after the filtering.

Reply:
that works, I think I took the smoothed version because in my mind it was less likely to have missing data (but in reality, both series should have the dates we need)